src/extension.ts

```diff
@@ -1275,14 +1280,20 @@ async function reportDeleteResult(resourceId: string, shellResult: ShellResult)
     refreshExplorer();
 }
 
-const deleteKubernetes = async (explorerNode?: explorer.ResourceNode) => {
+async function deleteKubernetes(delMode: KubernetesDeleteMode, explorerNode?: explorer.ResourceNode) {
+
+    let delModeString: string = '';
```

Comment:
Minor: consider using a ternary expression or helper function so that `delModeString` can be `const`.